docs/kamaji-azure-deployment-guide.md

```diff
@@ -102,24 +121,36 @@ metadata:
 spec:
   controlPlane:
     deployment:
-      replicas: 2
+      replicas: 3
       additionalMetadata:
-        annotations:
-          environment.clastix.io: ${TENANT_NAME}
         labels:
           tenant.clastix.io: ${TENANT_NAME}
-          kind.clastix.io: deployment
+      extraArgs:
+        apiServer: []
+        controllerManager: []
+        scheduler: []
+      resources:
+        apiServer:
+          requests:
+            cpu: 500m
+            memory: 512Mi
```

Comment:
Let's mention also the limits, so the end-users are aware of this option that can fine-tune according to their needs.

Reply:
done